tst/RtcpFunctionalityTest.cpp

```diff
@@ -471,6 +471,44 @@ TEST_F(RtcpFunctionalityTest, updateTwccHashTableTest)
     EXPECT_EQ(STATUS_SUCCESS, freePeerConnection(&pRtcPeerConnection));
 }
 
+TEST_F(RtcpFunctionalityTest, updateTwccHashTableIntPromotionCase) {
+    PRtcPeerConnection pRtcPeerConnection = NULL;
+    PKvsPeerConnection pKvsPeerConnection = NULL;
+    RtcConfiguration config{};
+    EXPECT_EQ(STATUS_SUCCESS, createPeerConnection(&config, &pRtcPeerConnection));
+    pKvsPeerConnection = reinterpret_cast<PKvsPeerConnection>(pRtcPeerConnection);
+    PTwccRtpPacketInfo pTwccRtpPacketInfo = NULL;
+    INT64 duration = 0;
+    UINT64 receivedBytes = 0, receivedPackets = 0, sentBytes = 0, sentPackets = 0;
+    UINT16 i;
+
+    // Grab the hash table
+    PHashTable pTwccRtpPktInfosHashTable = pKvsPeerConnection->pTwccManager->pTwccRtpPktInfosHashTable;
+    UINT16 hashTableInsertionCount = 0;
+
+    // Set up the hash table
+    pKvsPeerConnection->pTwccManager->prevReportedBaseSeqNum = UINT16_MAX;
+    pKvsPeerConnection->pTwccManager->lastReportedSeqNum = UINT16_MAX;
```

Comment:
may this be named as
 `#define  TWCC_SEQ_NUM_MAX UINT16_MAX` and
`#define TWCC_SEQ_NUM_ROLLOVER (TWCC_SEQ_NUM_MAX + 1)`

Reply:
Yes we can adjust the naming in a future PR.